Gifski/ShowBehavior.swift

```diff
@@ -0,0 +1,38 @@
+struct ShowBehavior {
+	enum Option {
+		case always
+		case once
+	}
+
+	static func always(identifier: String) -> ShowBehavior {
+		return .init(identifier: identifier, option: .always)
+	}
+
+	static func once(identifier: String) -> ShowBehavior {
+		return .init(identifier: identifier, option: .once)
+	}
+
+	let identifier: String
+	let option: Option
+
+	private var key: Defaults.Key<Int> {
+		return Defaults.Key<Int>("__showBehavior_" + identifier, default: 0)
```

Comment:
```suggestion
		return Defaults.Key<Int>("__Tooltip__\(identifier)", default: 0)
```

Reply:
Why is it an Int, can't we just make it a Bool of whether it has been shown or not?

Reply:
I guess I was just being proactive about what the `ShowBehavior` can become in the future and it's not that big of an overhead to use count instead of a flag.